package.json

```diff
@@ -29,11 +29,19 @@
   },
   "devDependencies": {
     "babel-eslint": "^7.2.3",
+    "chai": "^4.1.0",
+    "chai-as-promised": "^7.1.1",
     "eslint": "^4.2.0",
+    "jsdom": "^11.1.0",
+    "mocha": "^3.4.2",
```

Comment:
Ah, ok, thanks. I assumed since we were using mocha there must be a mocha binary ... oops.